### ISSUE TITLE
Add clear documentation of the defined settings and how to set them [ENHANCEMENT]

### ISSUE THREAD

**Issue by mbakker7** [pastas/pastas#408]
It is difficult to find what setting are available and what they do: 'prec', 'evap' (not quite sure that is what they are called).
Maybe add a Notebook to find this back easily.

**Comment by raoulcollenteur** [comment]
Also related to Issue #455, the question is how to set the settings for stresses. With the changes made in PR #459, only pandas.Series are allowed as input to Stressmodels. Thus, the stress models are the only point where the settings can be changed now. I distinguish below between 1) new and 2) existing stress models.

## 0. Options for TimeSeries settings

Here are the settings that are used in the TimeSeries objects, along with the default values:

```
self.settings = {
    "freq": None,
    "sample_up": None,
    "sample_down": None,
    "fill_before": None,
    "fill_after": None,
    "tmin": series.index.min(),
    "tmax": series.index.max(),
    "time_offset": pd.Timedelta(0),
}
```

From these,  `freq`, `tmin`,`tmax`, and `time_offset` are determined automatically from the provided time series themselves.

## 1. For new stress models

**Option 1 for new stress models:**

This is the preferred way to adjust/set the settings, directly upon creating of the stressmodel. Perhaps the easiest way is to select on of the predefined settings as follows: 

```
sm = ps.Stressmodel(stress, settings="prec")
```

In the above code the settings from `ps.rcParams["timeseries"]["prec']` are used. One can check `ps.rcParams["timeseries"]` for what that means and what other options are available.

**Option 2 for new stress models:**

If a user requires more control over the settings, the second option is to provide a dictionary with the settings.

```
settings = {
  "fill_before": 0.0,
  "fill_after": 0.0,
  # Etcetera
}
sm = ps.Stressmodel(stress, settings=settings)
```

I think this should be clear. 

## 2. For existing stress models

This is more complicated, except for `tmin`,`tmax`, and `freq`, which can be set through `sm.update_stress` for all stresses at the same time. There is currently no method to properly set `sample_up`, `sample_down`, `fill_before`, and `fill_after` after creation of a stressmodel. In this case, the only way is to access the stress TimeSeries object directly (`sm.stress[0].update_series`).

**Comment by raoulcollenteur** [comment]
@mbakker7, please check if this is included in the current docs for 1.0, and close this issue if this is fine now :)

It is reported here:
- https://pastas--502.org.readthedocs.build/en/502/userguide/02_time_series.html
- https://pastas--502.org.readthedocs.build/en/502/userguide/05_stressmodels.html
- https://pastas--502.org.readthedocs.build/en/502/api/generated/pastas.timeseries.TimeSeries.html

I will add to the doctoring of each stressmodel how to see the predefined settings as well now.

**Comment by mbakker7** [comment]
Looks nice now.
